x/axelarnet/keeper/genesis.go

```diff
@@ -9,6 +9,7 @@ import (
 func (k Keeper) InitGenesis(ctx sdk.Context, nexus types.Nexus, genState *types.GenesisState) {
 	k.setParams(ctx, nexus, genState.Params)
 	k.SetFeeCollector(ctx, genState.CollectorAddress)
```

Comment:
Not panic if error is returned?

Reply:
I wanted to throw a panic, but then somebody convinced not to do it. But I'll had it back